### DISCUSSION TITLE
Error email scope

### DISCUSSION THREAD

**Discussion by ryzam** [ory/hydra#2917] [Q&A] [answered]
HI, I try to integrate hydra with our internal development project using oidc.js. Our hydra running on docker based on the tutorial from hydra website. Unfortunately I'm not able to get email claim from the token

![image](https://user-images.githubusercontent.com/597759/148330590-486aa9a2-a373-498f-a433-10b38f2f5923.png)

**Comment by vinckr** [comment, marked as answer]
Hello @ryzam,
We need a bit more information to be able to help. 

What flow are you performing? 
Are you running Hydra just like in the quickstart, or what did you change?

**Reply by ryzam** [reply]
Hi @vinckr 

Sorry for my late response, I used code flow and fully follow step by step from quicstart.
Any other logs/information needed? I'll try to supply later.

**Reply by vinckr** [reply]
Hey @ryzam , does your problem still persist? 
I will mark this as answered for now, but feel free to come back to the discussion any time.
